aws-console-redactor.user.js

```diff
@@ -23,6 +23,14 @@ watchForElements(queryAccountId, "accountId", function (element) {
   redactPattern(accountId); // Redact by account ID
   redactPattern("arn:aws"); // Redact all ARNs
   redactPattern("Account:"); // Redact trusted accounts on IAM roles list view
+  
+  try {
+        const accInheader = accountId.substring(0,4) + "-" + accountId.substring(4,8) + "-" + accountId.substring(8);
```

Comment:
Is there any situation where this section will actually raise an exception?

Since AccountIDs always have 12 digits, can we get rid of the try/catch?

Reply:
Makes sense, we can get rid of it. Will update it 👍